src/libs/portfolio/getOnchainBalances.ts

```diff
@@ -176,14 +176,23 @@ export async function getNFTs(
       : null
 
     const token = mapToken(beforeToken)
+    const [receiving, sending]: [bigint[], bigint[]] = [[], []]
```

Comment:
why not simply: const receiving: bigint[] = []; const sending: bigint[] = []

Reply:
wanted to minimize non-logic bearing lines, but ok